components/containers/partitioned_vector/include/hpx/components/containers/partitioned_vector/partitioned_vector_decl.hpp

```diff
@@ -12,8 +12,8 @@
 
 #include <hpx/config.hpp>
 #include <hpx/assertion.hpp>
-#include <hpx/lcos/wait_all.hpp>
-#include <hpx/lcos/when_all.hpp>
+#include <hpx/distributed_lcos/wait_all.hpp>
+#include <hpx/distributed_lcos/when_all.hpp>
```

Comment:
The `wait_all` and `when_all` facilities are purely local (so are `wait_any`, `wait_n`, `when_any`. and `when_n`)

Reply:
The are also in àsync_combinators` in #4478 (which I will merge today if CI looks good).

Reply:
ok thanks !